### ISSUE TITLE
🐞 [Bug Report] - Tutorial references unused Container Registry

### ISSUE THREAD

**Issue by carlosaenz26** [GoogleCloudPlatform/avocano#414]
Hey guys, 
as metioned in the other issue I ran 

```
gcloud artifacts repositories create containers \
    --repository-format=docker \
    --location=us
```

after doing so in the 3 step, one must Update this job to use the image I just built, which includes my code changes.  so I ran 
```
gcloud run jobs update \
    migrate${DEPLOY_SUFFIX} \
    --image \
    gcr.io/${PROJECT_ID}/server \
    --region $REGION
```
but I got this error:
```
 X  Updating job... Image 'gcr.io/myprojectid/server' not found.                                                                                                                                      
Job failed to deploy
```
I understand that the image is now in the containers right? if so how in the job do specify that? 

please help

**Comment by glasnt** [comment]
Hi @carlosaenz26. It looks like you found another inconsistency with the migration of this sample from Container Registry to Artifact Registry. I'll make sure this is addressed along with the issue in #412. 

(ps it's useful to report issues you experience on repos, but please refrain from both commenting on an existing issue and creating a new one)

**Comment by glasnt** [comment]
This should now be resolved.
